events.py

```diff
@@ -19,30 +19,16 @@ def clear(bot, event, *args):
 
 
 def join(bot, event, *args):
-    """<b>/join <id></b> join the event with given id.
-Leave the id blank to join the latest created event.
-Alternatively use /event join <id>."""
+    """<br/>See event join."""
```

Comment:
Don't you think duplicating `Join the event <id>. Omitting <id> will join the latest created event.` is a good idea here?